build.yaml

```diff
@@ -0,0 +1,9 @@
+builders:
+  sass_builder:
+    target: "sass_builder"
```

Comment:
Note that this is a fundamental change to how this builder works today, because without the `build_to: source` option it will default to writing to the cache.

This is actually what users of the sass builder will ultimately want, but it does mean that you will need to go through the `pub run build_runner serve` command to serve your app. Presumably users of sass_builder are already off of pub though - or are not using it for transformer purposes.

The reason this new behavior is actually better, is that it will compile sass for all dependencies, not just the current package. This is in contrast to having all packages publish their compiled css files which is not actually the correct thing to do in all cases (they depend on the exact versions of your dependencies at the time, which might be different for your consumers).

Reply:
The nice thing about this is that it brings users of the builder inline with users of the Transformer.

Reply:
running the example, when I change the `target` to`source` it still does not generates the files. What should it be the configuration?

Reply:
It isn't the `target` you'd want to change, it's the `build_to` (which is not present here since we're using the default)

Add `build_to: source` as a key under `sass_builder:` to see this in action. Note that this isn't something users can override themselves without duplicating the entire `build.yaml` as `sass_builder.build.yaml`

Reply:
I have no luck yet, could you show me some better example

Reply:
Do you have a need to publish a merged directory of the source and generated output? This isn't safe to do if you have dependencies on other packages and import their sass files in your package like in the example from this package.

Here is an updated example forcing the output into the source directory: https://github.com/nshahan/sass_builder/tree/update-example/example

This approach is not recommended as a maintainable solution but can be helpful when debugging. It overrides the build config for any uses of sass_builder in all of your dependencies.

If you want to capture the output from the build you should use:
```
pub run build_runner build --output path/to/output/directory
```

__This will delete the contents of your output directory.__